src/mavsdk_server/src/CMakeLists.txt

```diff
@@ -43,10 +44,15 @@ else()
     add_library(mavsdk_server ${MAVSDK_SERVER_SOURCES})
 endif()
 
+if(BUILD_WITH_PROTO_REFLECTION)
```

Comment:
Just wondering: instead of having that split in 3 places, what about simply adding another `target_link_libraries` below the current one? Like this:

```cmake
if(BUILD_WITH_PROTO_REFLECTION)
    target_link_libraries(mavsdk_server
        PRIVATE
        gRPC::grpc++_reflection
    )
endif()
```

Feels more direct/clearer to me. Do you have an opinion?

Reply:
Agree with you there :) and updated accordingly.